web/src/components/pages/profile/avatar-setup/avatar-setup.css

```diff
@@ -26,9 +26,23 @@
         left: 15%;
         right: 15%;
         padding-top: 10px;
-        @media only screen and (max-width: 375px) {
+
+        @media only screen and (max-width: 415px) {
             left: 6%;
             right: 6%;
+            display: none;
```

Comment:
I don't think we want to actually hide the recording on mobile in general. Rather only on browsers that don't support recording